test/binary/test_61.kore

```diff
@@ -0,0 +1,2 @@
+// RUN: %strip-binary -i %test-input -k -o %t.out
+// RUN: diff %t.out %test-diff-out
```

Comment:
Perhaps add a comment here explaining why we're testing like this, and with large files specifically.

Reply:
Done!